app/models/concerns/submittable/solvable.rb

```diff
@@ -12,4 +14,20 @@ def run_tests!(params)
         locale: locale,
         expectations: expectations))
   end
+
+  private
+
+  def serialize(content)
```

Comment:
`join_files` or `merge_files`, perhaps?  The term `serialize` is being used along the application with a more specific meaning - translating compound values into a JSON or YAML - and It is a little confusing here.